Runtime/Interaction/SnapZone.cs

```diff
@@ -76,7 +76,6 @@ public Material HighlightMeshMaterial
 
                 return highlightMeshMaterial;
             }
-            set { highlightMeshMaterial = value; }
```

Comment:
Why was this removed?

Reply:
dunno, you removed it in your PR and it was not used :x

Reply:
🤔🤷‍♂️

Reply:
No idea, but if this was already published, this is an API break.